tests/queries/0_stateless/03166_from_readable_size.sql

```diff
@@ -0,0 +1,31 @@
+-- Should be the inverse of formatReadableSize
+SELECT formatReadableSize(fromReadableSize('1 B'));
+SELECT formatReadableSize(fromReadableSize('1 KiB'));
+SELECT formatReadableSize(fromReadableSize('1 MiB'));
+SELECT formatReadableSize(fromReadableSize('1 GiB'));
+SELECT formatReadableSize(fromReadableSize('1 TiB'));
+SELECT formatReadableSize(fromReadableSize('1 PiB'));
+SELECT formatReadableSize(fromReadableSize('1 EiB'));
+
+-- Should be the inverse of formatReadableDecimalSize
+SELECT formatReadableDecimalSize(fromReadableSize('1 B'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 KB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 MB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 GB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 TB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 PB'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 EB'));
+
+-- Is case-insensitive
+SELECT formatReadableSize(fromReadableSize('1 mIb'));
+SELECT formatReadableDecimalSize(fromReadableSize('1 mb'));
+
+-- Should be able to parse decimals
```

Comment:
What is missing are "negative" tests, i.e. tests that check that certain exceptions are thrown for invalid inputs:
- too few / too many arguments
- the right number of arguments but of invalid type (e.g. integer arguments)
- the right number and type of arguments but garbage inputs (e.g. `'dkfdl 10. skdk 0d'`)

Grep for `serverError` in tests/queries/0_stateless to find the right syntax to check for specific exception types.

Also, please check that these inputs work too:
- `NULL` inputs (see `toNullable`)
- dictionary-encoded inputs (see `toLowCardinality`)
- non-const inputs (see `materialize`)